manifests/base/redis-lb.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: cm-redis-lb
+spec:
+  type: LoadBalancer
```

Comment:
Do you need to access this from elsewhere?  Maybe a ClusterIP is sufficient?

Reply:
The deployment on this PR is to facilitate running the FastAPI part of the service in debug mode "in-hand" on rubin dev nodes, with just redis and postgres in k8s.  Safir needs access to redis, thus the lb.